source/sixve.f90

```diff
@@ -61,26 +61,26 @@ subroutine sumpos
     write(lout,10010) nint(dlost),d(3),d(5),d(7),d(9),d(10),d(11),d(12),nint(d(16)),nint(d(18)),    &
       d(19),d(21),ch,d(4),d(6),d(8),d(13),nint(d(17)),d(20),d(25),d(14),d(15)
   end do
-  20 continue
-  rewind 10
 
+  20 continue
```

Comment:
Why goto label and not a clean loop exit?

Reply:
Sorry, but I dom't have the time to rewrite every single subroutine i touch.

Reply:
#859